ext/openssl/ossl_bn.c

```diff
@@ -1217,14 +1248,15 @@ Init_ossl_bn(void)
 
     rb_define_method(cBN, "initialize_copy", ossl_bn_copy, 1);
     rb_define_method(cBN, "copy", ossl_bn_copy, 1);
+    rb_define_method(cBN, "dup", ossl_bn_dup, 0);
```

Comment:
`BN` implements `#initialize_copy`. The `#dup` method defined by `Object` should already work without this.

Reply:
Will resolve with a comment to that effect so others know as well

Reply:
Should we alias `+@` to `#dup` then?

Reply:
I think the current state (having `#+@` as a normal method defined by `ossl_bn_uplus()`) is fine.